### ISSUE TITLE
Paramaters not used in nltk.text.concordance()

### ISSUE THREAD

**Issue by BLKSerene** [nltk/nltk#2071]
nltk.text.concordance() call print_concordance() and pass word, width, lines as parameters, but print_concordance() then call find_concordance(word, width=80, lines=25) where parameter width and lines are assigned 80 and 25, rendering the parameters input by the user useless (when printing the concordance results, there are always 25 lines whose width are 80-charater long).

**Comment by alvations** [comment]
This should have been fixed with #2026 in the bleeding edge branch. 

If you want it to be resolved now, you can try the bleeding edge version with the following:

```
pip3 install -U https://github.com/nltk/nltk/archive/develop.zip 
```

Otherwise waiting for the next version is the other option.

**Comment by alvations** [comment]
Resolved in #2026
